src/routes/docs/products/sites/quick-start/tanstack-start/+page.markdoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: article
+title: Deploy a Tanstack Start app to Appwrite Sites
+description: Learn how to setup and deploy Tanstack Start apps on Appwrite Sites.
+---
+
+{% section #step-1 step=1 title="Create Tanstack Start app" %}
+
+First, you must either create a Tanstack Start app or setup the [Tanstack Start starter template](https://github.com/appwrite/templates-for-sites).
+
+Open your terminal, and run the following command.
+
+```bash
+npm create @tanstack/start@latest
+```
+
+Push this project to a [GitHub repo](https://github.com/new).
+{% /section %}
+
+{% section #step-2 step=2 title="Create Appwrite project" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io).
+
+{% only_dark %}
+![Create Appwrite project](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create Appwrite project](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+If this is your first time using Appwrite, create an account and create your first project.
+{% /section %}
+
+{% section #step-3 step=3 title="Create site" %}
+Head to the **Sites** page in your Appwrite project, click on the **Create site** button, and select **Connect a repository**.
+
+Connect your GitHub account and select the repository you intend to deploy.
```

Comment:
Might be worth mentioning there is option to just allow all. In theory, that is beneficial for us. So they dont need to re-configure in future